.changeset/lovely-foxes-bow.md

```diff
@@ -0,0 +1,14 @@
+---
+'@shopify/cli-kit': minor
+'@shopify/create-hydrogen': minor
```

Comment:
Not that aware of the CLI versioning but since there is a git requirement now. should this be a `major`? if `git` was already required by the CLI then should be fine a `minor`

Reply:
Git was already a requirement.